api-extractor.json

```diff
@@ -0,0 +1,364 @@
+/**

```

Comment:
just out of curiosity are there differences between this file and the api-extractor.json in answers-core? I remember you or Connor mentioning something about a config option "just working"

Reply:
yes, the difference is primarily the `bundledPackages`. here, we are adding answers-core as a bundled package, which adds it directly to the generated bundle for headless. this solves the problem of links not resolving because they are for types/interfaces in an external package